### ISSUE TITLE
Install and run react-heatmap with react 16.x

### ISSUE THREAD

**Issue by gukii** [JonathanWi/react-heatmap#15]
I tried to install react-heatmap with react 16.4, and lower 16. versions, but can t get it to run. 

Has anybody had success in using react-heatmap with a react 16.x project? 

Please share some insights on how u got it to work.. Thx!

**Comment by Lynnhan95** [comment]
No, it doesn't support React higher version (16.), saw an proType issue everytime.

**Comment by luizcarlospedrosogomes** [comment]
besides updating by hand. is there an alternative?
